boards/arm/adafruit_kb2040/adafruit_kb2040.dts

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2021 Yonatan Schachter
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <rpi_pico/rp2040.dtsi>
+#include "adafruit_kb2040-pinctrl.dtsi"
+#include "sparkfun_pro_micro.dtsi"
+#include <freq.h>
+
+/ {
+	chosen {
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,console = &uart0;
+		zephyr,code-partition = &code_partition;
+	};
+
+	xtal_clk: xtal-clk {
+		compatible = "fixed-clock";
+		clock-frequency = <12000000>;
+		#clock-cells = <0>;
+	};
+};
+
+&flash0 {
+	/* 8MB of flash minus the 0x100 used for
```

Comment:
This can be removed (this is an old comment from the pico which is no longer relevant)

Reply:
Done.